### PR TITLE
Update Advanced Settings documentation

### PR DESCRIPTION
**Pull request by olliebennett**
I know this page has recently been worked on and I hope my changes aren't too interfering. Feel free to select or cherry-pick a subset of changes if you prefer.

On reading the Advanced Settings section of the PhotoPrism app, there were still areas which weren't clear to me, in particular around how the size limits worked. I have slightly restructured the corresponding documentation to clarify this.

### CODE REVIEW

File: docs/user-guide/settings/advanced.md [hunk 1]
```diff
@@ -1,92 +1,161 @@
...
-## Options ##
+This page enables advanced control of PhotoPrism,
+and configuring image and thumbnail handling.
 
-#### Debug Logs ####
+![](img/advanced-settings.jpg)
+
+The same options can all be controlled instead using their corresponding
+[config options](/getting-started/config-options/).
+Settings updated here will override values already configured.
```
Comment (olliebennett) [inline docs/user-guide/settings/advanced.md:10]: >  Settings updated here will override values already configured.

Is this true? I assumed the options set on this page take priority over any environment variables or command flags.

File: docs/user-guide/settings/advanced.md [hunk 2]
```diff
@@ -1,92 +1,161 @@
...
+As an example of file size, if a quality of 95 results in a preview file size of 500kB,
+reducing quality to 80 reduces that file size down to about 100kB.
+
+Image classification (which passes the thumbnails through TensorFlow)
+also works better with sharp images,
+so it's likely a lower quality will result in less accurate labels.
+
+The equivalent [config option](/getting-started/config-options/) is `PHOTOPRISM_JPEG_QUALITY`.
+
+### Dynamic and Static Size Limits
```
Comment (olliebennett) [inline docs/user-guide/settings/advanced.md:113]: Please verify this section; it's mostly new content.

File: docs/user-guide/settings/advanced.md (hunk 1 above)
Comment (lastzero) [reply docs/user-guide/settings/advanced.md:10]: These values will be stored in the `config` folder and override whatever was configured before e.g. via environment variables. Otherwise, making changes wouldn't make much sense.

File: docs/user-guide/settings/advanced.md [hunk 3]
```diff
@@ -1,92 +1,161 @@
...
+Requires restart for changes to be applied.
 
-#### Disbale ExifTool ####
-This option prevents the creation of json files with exif data in `storage/sidecar`. 
+The equivalent [config toggle](/getting-started/config-options/) is `PHOTOPRISM_DISABLE_WEBDAV`.
 
-#### Disable TensorFlow ####
-When selected, no labels are created for your files.
+### Disable Places
+When selected, no geo-information is read from your files metadata.
```
Comment (lastzero) [inline docs/user-guide/settings/advanced.md:48]: Lat/long will be indexed, but not resolved via reverse lookups to locations (country, state, city, location type,...)

File: docs/user-guide/settings/advanced.md [hunk 4]
```diff
@@ -1,92 +1,161 @@
...
-right_224 | 224    | 224     | TensorFlow               |
-tile_500  | 500    | 500     | Cards Preview            |
-fit_720   | 720    | 720     | Mobile, TV               |
-fit_1280  | 1280   | 1024    | Mobile, HD Ready TV      |
-fit_1920  | 1920   | 1200    | Mobile, Full HD TV       |
-fit_2048  | 2048   | 2048    | Tablets, Cinema 2K       |
-fit_2560  | 2560   | 1600    | Quad HD, Retina Display  |
-fit_3840  | 3840   | 2400    | Ultra HD                 |
-fit_4096  | 4096   | 4096    | Ultra HD, Retina 4K      |
-fit_7680  | 7680   | 4320    | 8K Ultra HD 2, Retina 6K |
```
Comment (lastzero) [inline docs/user-guide/settings/advanced.md:68]: Why did you delete the table showing preview image sizes? Seems very important to understand what files will be created based on the settings...

**Comment by lastzero** [comment]
Thank you so much! Answered your questions.

You seem to have removed the complete preview image size table? Don't you think it is helpful?

File: docs/user-guide/settings/advanced.md [hunk 5]
```diff
@@ -173,4 +242,13 @@ In addition, PhotoPrism also supports TIFF, PNG, BMP and GIF files. Be aware tha
...
+    RAW to JPEG conversion also works in read-only mode.
+
+### JPEG Size Limit
+
+This controls the maximum size of downsampled JPEG preview files to create
+when converting original RAW images.
+
+### Use Presets
+
+Disables simultaneous conversion of RAW files to apply Darktable presets.
```
Comment (olliebennett) [inline docs/user-guide/settings/advanced.md:254]: I wasn't sure what this really means.

I think:

> PhotoPrism uses [Darktable](https://www.darktable.org/) behind the scenes to convert RAW images to JPEG. 

But I don't understand what the "preset values" are or means. What happens if I don't tick this box? What would be used instead of "presets"?

Comment (lastzero) [reply docs/user-guide/settings/advanced.md:254]: Defaults

Comment (olliebennett) [reply docs/user-guide/settings/advanced.md:256]: Okay, thanks. Are the defaults configurable or viewable anywhere? And where can I set the presets to override the defaults (if at all)?

File: docs/user-guide/settings/advanced.md (hunk 3 above)
Comment (olliebennett) [reply docs/user-guide/settings/advanced.md:48]: Brilliant, thanks. I'll update to clarify this.

File: docs/user-guide/settings/advanced.md (hunk 4 above)
Comment (olliebennett) [reply docs/user-guide/settings/advanced.md:68]: The list of thumbnail sizes is listed further down the page and this information is available via the 'developer docs' section which I've linked to, in case users wanted to dig deeper. Happy to restore this table if you like though. I do see some benefit for users knowing which files will be used where.

**Comment by olliebennett** [comment]
Thanks a lot for the feedback. I've restored the table; you're right that it's helpful. I've found a way to reduce duplication by removing the list of images and combining that via a comment next to the table.

I've also pushed updates to reflect your other responses.

File: docs/user-guide/settings/advanced.md (hunk 5 above)
Comment (lastzero) [reply docs/user-guide/settings/advanced.md:262]: Good question, it's actually more a question about Darktable than PhotoPrism. A contributor told us to add this option so that converting RAW files is faster.

File: docs/user-guide/settings/advanced.md [hunk 6]
```diff
@@ -1,53 +1,156 @@
...
+This will allow the server to generate downsampled preview images on-demand,
+causing a delay when previewing a photo in full screen mode.
+
+#### What files will be created by PhotoPrism?
+
+The minimum size for static previews is 720px, so all images up to `720x720` will be rendered.
+Higher static limits will additionally generate previews only up to the defined limit.
+
+The files are generated in the `storage/cache/thumbnails` folder,
+with file paths dependent on the size and original file's hash, such as;
```
Comment (lastzero) [inline docs/user-guide/settings/advanced.md:149]: such as:

File: docs/user-guide/settings/advanced.md (hunk 5 above)
Comment (lastzero) [reply docs/user-guide/settings/advanced.md:262]: Testing how exactly this affects image style & quality is on our todo.

File: docs/user-guide/settings/advanced.md (hunk 6 above)
Comment (olliebennett) [reply docs/user-guide/settings/advanced.md]: Indeed! Fix pushed; thanks.

**Comment by lastzero** [comment]
Thank you so much! Merged this, you're welcome to send more fixes / improvements if needed.
